txpipe/twopoint.py

```diff
@@ -331,12 +378,24 @@ def get_shear_catalog(self, data, i):
         import treecorr
         g1,g2,mask = self.get_m(data, i)
 
-        cat = treecorr.Catalog(
-            g1 = g1,
-            g2 = g2,
-            ra = data['ra'][mask],
-            dec = data['dec'][mask],
-            ra_units='degree', dec_units='degree')
+        if self.config['var_methods']=='jackknife':
+            patchcenters = self.get_input('patch_centers')
```

Comment:
Two of the occurrences of "patchcenters" have underscores between "patch" and "centers", yet the third does not.  Can you harmonize by changing to patch_centers? (am feeling very pedantic today)